.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 coverage
 npm-debug.log
 
+/demo/src/app/share/
```

Comment:
With my PR #32, i think all changes made to demo project  in this commit, can be reverted.